src/components/AreaCard/AreaCard.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
+import ListingsContainer from '../ListingsContainer/ListingsContainer';
+import { Link } from "react-router-dom";
 import './AreaCard.scss';
 
-function AreaCard({ id, key, name, shortname, about }) {
+function AreaCard({ id, name, shortname, about }) {
   return <article className="area-card">
       <section>
         <h1>{name}</h1>
         <h4>({shortname})</h4>
       </section>
       <p>{about}</p>
-        <button>View Listing</button>
+      <Link to={"/areas/" + id}>
+        <button>View Listings</button>
+      </Link>
     </article>
```

Comment:
This is a good set up to link the page route to a listing page based off of the listing ID.  This can make it clear of what listing is presented in the app url.